components/cloud-foundry/src/services/app-wall-actions.service.js

```diff
@@ -0,0 +1,58 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('cloud-foundry.service')
+    .factory('cfAppWallActions', appWallActions)
+    .run(register);
+
+  /* eslint-disable no-unused-vars */
+  // Ensure that an instance of cfAppWallActions is created by injecting it here.
+  function register(cfAppWallActions) { }
```

Comment:
I've nuked it, because cfAppWallActions is pulled in elsewhere it's not needed here